BasicSample/Assets/Shared/Scripts/DisconnectAppRemoting.cs

```diff
@@ -0,0 +1,11 @@
+using UnityEngine;
+using Microsoft.MixedReality.OpenXR.Remoting;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    // Helper script for hooking MRTK/Unity events to the AppRemoting Disconnect method.
+    public class DisconnectAppRemoting : MonoBehaviour
```

Comment:
why are these in "shared" folder instead of the "remoting" sample folder?

Reply:
good call; just moved these when tidying up the RemotingSample scene!